src/executor/evaluate/evaluated.rs

```diff
@@ -99,7 +101,7 @@ impl<'a> PartialOrd for Evaluated<'a> {
                 ValueRef(r) => r.partial_cmp(l).map(|o| o.reverse()),
                 Value(r) => r.partial_cmp(*l).map(|o| o.reverse()),
```

Comment:
fyi.
Reason that `reverse()` is used is that this is comparison between `gluesql::data::Value` and `sqlparser::ast::Value`.

`gluesql::data::Value` has implementation of `PartialCmp` with `sqlparser::ast::Value`.
However `sqlparser::ast::Value` does not have implementation of `PartialCmp` with `gluesql::data::Value`.
And Rust does not allow adding more `impl` in different modules.
so that's why `reverse` was required.